bin/ConcordanceCheck.sh

```diff
@@ -250,7 +250,11 @@ fi
 			exit 1
 			}
 
-	echo "${concordanceCheckVersion}" > "${concordanceDir}/results/${concordanceCheckId}.ConcordanceCheckVersion"
+	# Adding concordance pipeline version into .sample file.
+	echo -e "concordanceCheckVersion\n${concordanceCheckVersion}" > "${concordanceDir}/results/${concordanceCheckId}.ConcordanceCheckVersion"
+	paste "${concordanceDir}/results/${concordanceCheckId}.sample" "${concordanceDir}/results/${concordanceCheckId}.ConcordanceCheckVersion" > "${concordanceDir}/results/${concordanceCheckId}.tmp"
+	mv -v "${concordanceDir}/results/${concordanceCheckId}."{tmp,sample}
+	rm "${concordanceDir}/results/${concordanceCheckId}.ConcordanceCheckVersion"
```

Comment:
```suggestion
	# Add ConcordanceCheck pipeline version to *.sample file.
	printf '\nconcordanceCheckVersion\n%s\n' "${concordanceCheckVersion}" >> "${concordanceDir}/results/${concordanceCheckId}.sample"
```

Reply:
Dat toevoegen van de versie van de ConcordanceCheck pipeline kan makkelijker met `printf` en rechtstreeks toevoegen aan bestaand bestand. Het _format_ voor `printf` is een beetje een gok. Ik weet niet of je eerst een `\n` moet toevoegen of dat dat er al in zat. Kortom, mogelijk heb je genoeg aan
`'concordanceCheckVersion\n%s\n'`

Reply:
beide suggesties werken niet, schrijft onderaan de file regels bij ipv extra kolom achteraan de file wat hier gebeurd. Krijg het met awk met een dikke oneliner wel in 2 regels voor elkaar maar wordt er niet leesbaarder van. Dit werkt op zich prima toch

Reply:
ik zou het persoonlijk wel met awk doen hoor:
awk -v c="${concordanceCheckVersion}" '{if (NR>1){print $0"\t"c}else{print $0"\tConcordanceCheckVersion"}}' "${concordanceDir}/results/${concordanceCheckId}.sample" > "${concordanceDir}/results/${concordanceCheckId}.sample.tmp"
mv "${concordanceDir}/results/${concordanceCheckId}.sample{.tmp,}

heb je ook geen "last" van die extra ConcordanceCheckVersion file die je nog extra maakt.

Reply:
aangepast